commit c334125fbbb7994d6bcab8ecf735bf19a2ef1444
Author: Martin Gerhardy <martin.gerhardy@gmail.com>
Date:   Wed Jun 25 11:06:25 2014 +0200

    * hopefully fixed bug #5336 (active c++11 for mac only to fix a compile problem until we sort out the windows build chain issues)
    
    * Compile error on Mac OS 10.9: src/client/cl_http.cpp:192:53: error: cannot pass object of non-POD type 'std::__1::nullptr_t' through variadic function

diff --git a/build/platforms/darwin.mk b/build/platforms/darwin.mk
index 89a20a5806..8dee5e3138 100644
--- a/build/platforms/darwin.mk
+++ b/build/platforms/darwin.mk
@@ -4,6 +4,7 @@ SO_LDFLAGS                = -dynamiclib
 SO_LIBS                  := -ldl
 
 CFLAGS                   += -D_BSD_SOURCE -D_XOPEN_SOURCE
+CXXFLAGS                 += -std=c++11
 LDFLAGS                  += -framework IOKit -framework Foundation -framework Cocoa -headerpad_max_install_names
 
 ### most mac users will have their additional libs and headers under /opt/local
